pkg/cmd/roachtest/tests/failure_injection.go

```diff
@@ -599,6 +613,97 @@ var dmsetupDiskStallTest = func(c cluster.Cluster) failureSmokeTest {
 	}
 }
 
+var nodeKillTests = func(c cluster.Cluster) []failureSmokeTest {
+	rng, _ := randutil.NewPseudoRand()
+	var tests []failureSmokeTest
+	for _, gracefulShutdown := range []bool{true, false} {
+		var killedNodeGroup, unaffectedNodeGroup option.NodeListOption
+
+		for {
+			groups, _ := c.CRDBNodes().SeededRandGroups(rng, 2 /* numGroups */)
+			killedNodeGroup = groups[0]
+			unaffectedNodeGroup = groups[1]
+
+			// If there is only one live node left, draining will hang indefinitely while
```

Comment:
A followup PR will bump the size of this cluster so we can test rebalancing ranges. So this is only temporary that we don't test multi node graceful drains.